src/debugger/Debugger.js

```diff
@@ -97,16 +107,17 @@ export class DebugServer {
       if (filePath === null) return;
       let lineNum = location.start.line;
       let colNum = location.start.column;
-
       // Check whether there is a breakpoint we need to stop on here
-      if (!this.proceedBreakpoint(filePath, lineNum, colNum)) return;
-
+      let breakpoint = this.findStoppableBreakpoint(filePath, lineNum, colNum);
+      if (breakpoint === null) return;
       // Tell the adapter that Prepack has stopped on this breakpoint
-      this.channel.writeOut(`breakpoint stopped ${lineNum}:${colNum}`);
+      this.channel.writeOut(
+        `${DebugMessage.BREAKPOINT} ${DebugMessage.BREAKPOINT_STOPPED} ${breakpoint.filePath} ${breakpoint.line}:${breakpoint.column}`
+      );
 
       // Wait for the adapter to tell us to run again
       this.waitForRun(function(line) {
-        return line === "proceed";
+        return line === DebugMessage.PREPACK_RUN;
```

Comment:
Please do not use "line" here which leaks the implementation details of we are using a file line, use command or message.